Formula/docbook.rb

```diff
@@ -77,6 +77,10 @@ def install
   end
 
   def post_install
+    on_linux do
+      ENV["PATH"] = "#{ENV["PATH"]}:#{Formula["libxml2"].opt_bin}"
```

Comment:
```suggestion
      ENV.prepend_path "PATH", Formula["libxml2"].opt_bin
```

Reply:
I tried `append_path` but got this error:

```
==> An exception occurred within a child process:
  NoMethodError: undefined method `append_path' for #<Object:0x000000000269f990>
```

Reply:
Same with `prepend_path`:

```
==> An exception occurred within a child process:
  NoMethodError: undefined method `prepend_path' for #<Object:0x00000000012bf9c0>
```

Reply:
We have 135 occurrences of `ENV.prepend_path` in homebrew-core, so that's scary. Can you investigate and see why it's failing for you?

Reply:
`prepend_path` isn't available during `postinstall`, will be resolved by https://github.com/Homebrew/brew/pull/10062